pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[project]
+name = "bson"
+version = "0.5.10"
+authors = [
+    { name = "Ayun Park", email = "iamparkayun@gmail.com" }
+]
+description = "BSON codec for Python"
+readme = "README.rst"
+requires-python = ">=3.9"
+dependencies = [
+    "python-dateutil>=2.9.0.post0",
+]
+license = {text = "BSD"}
```

Comment:
There are now *way* too many BSD licenses -- and I think "BSD" itself is not a valid SPDX identifier:

https://spdx.org/licenses/

I think the project should choose one.

And there is LICENSE, which looks non-standard to me :-( and APACHE_LICENSE, which is Apache-2.0

Also, in LICENSE, I see:
```
Copyright (c) 2010, Kou Man Tong
Copyright (c) 2015, Ayun Park
```
but only  Ayun Park is listed as an author here. which is OK -- the author list should reflect who's currently involved. Is Ayun Park still leading development? And Kou Man Tong not involved?

Also, there are more than one way to specify the license:

(from https://packaging.python.org/en/latest/guides/writing-pyproject-toml/)

This can take two forms. You can put your license in a file, typically LICENSE or LICENSE.txt, and link that file here:

```
[project]
license = {file = "LICENSE"}
```
or you can write the name of the license:
```
[project]
license = {text = "MIT License"}
```
If you are using a standard, well-known license, it is not necessary to use this field. Instead, you should use one of the [classifiers](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers) starting with License ::. (As a general rule, it is a good idea to use a standard, well-known license, both to avoid confusion and because some organizations avoid software whose license is unapproved.)